cases/PFEM_Metafor/elasticContainerFilling_container_Mtf.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python
+# -*- coding: latin-1; -*-
+
+''' 
+
+Copyright 2018 University of Liège
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+
+'''
```

Comment:
I think that the licence in python file is usually put as comments

Otherwise, it hides the useful documentation of the module/script

TODO... later...